tutorials/preprocessing/plot_80_boxy_processing.py

```diff
@@ -27,7 +27,12 @@
 
 boxy_data_folder = mne.datasets.boxy_example.data_path()
 boxy_raw_dir = os.path.join(boxy_data_folder, 'Participant-1')
-raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, 'AC', verbose=True).load_data()
+raw_intensity = mne.io.read_raw_boxy(boxy_raw_dir, 'Ph', verbose=True).load_data()
+
+### plot the raw data ###
+raw_intensity.plot(n_channels=10, clipping='clamp')
```

Comment:
I removed the parameter to add a new scaling to this plot so we can test default

Reply:
I assume the really noisy channels are one where the source is too far from the detector?

'clamp' helps with the noisy ones but maybe we shouldn't be doing that...

Reply:
yeah clamp hides data, i don't really like it, and the scaling should be fixed such that those are not as noisy looking any longer,

yeah really noisy ones are too far, so we can also assume people will remove those at an early step (ppod does not include these channels in its output for example)